elements/pfe-navigation/mock/site-switcher.html

```diff
@@ -66,7 +66,7 @@ <h3>Red Hat Partner Connect</h3>
                       </ul>
                       <!-- -->
 
-                      <hr>
+                      <hr aria-hidden="true">
```

Comment:
Iiinteresting, what do these things do for a screen reader?

Reply:
these things === `hr`

Reply:
@wesruv the screen reader says "horizontal splitter" but with the aria hidden set to true the screen readers just ignores hr tags completely which is why it is always best to do those lines with css rather than with the <hr> tag wherever possible.